virttest/qemu_vm.py

```diff
@@ -1507,6 +1510,9 @@ def sort_key(dev):
         devices.insert(StrDev('-S', cmdline="-S"))
         # Add the VM's name
         devices.insert(StrDev('vmname', cmdline=add_name(name)))
+        mouse_type = params.get('mouse_type', 'default')
+        if mouse_type != "default":
+            devices.insert(StrDev('mouse', cmdline=add_usbdevice(devices, mouse_type)))
```

Comment:
Usually, we add usb input devices by setting the usb related configuration, for instance, we have the following snippet:
https://github.com/avocado-framework/avocado-vt/blob/3d2b0feb51a42d29c6dcd679f43c716303581ea6/shared/cfg/base.cfg#L98-L102
it will be converted to:
```
    -device usb-tablet,id=usb-tablet1,bus=usb1.0,port=1  \
```
so supposes that you want usb mouse to be the default pointer device for all vt tests, the patch would be:
```patch
diff --git a/shared/cfg/base.cfg b/shared/cfg/base.cfg
index 87578f40..c8a4e617 100644
--- a/shared/cfg/base.cfg
+++ b/shared/cfg/base.cfg
@@ -96,10 +96,10 @@ usb_type_usb1 = ich9-usb-ehci1
 usb_max_port = 6
 
 # USB device object names (whitespace separated)
-usb_devices = tablet1
+usb_devices = mouse1
 # USB device type, run following command to see device list on your host.
 # `qemu-kvm -device \? 2>&1 | grep "bus USB"`
-usb_type_tablet1 = usb-tablet
+usb_type_mouse1 = usb-mouse
 # USB Controller type which device uses.
 usb_controller = ehci
 usb_bus = usb1.0
```

Also, we have another schema to add input devices (see https://github.com/avocado-framework/avocado-vt/pull/1737 for details):
https://github.com/avocado-framework/avocado-vt/blob/16f745b3d02188964a22d191b2d6ab572f8de172/shared/cfg/base.cfg#L683-L688
and it will support types other than `virtio` in future.

Reply:
Sounds perfect, our `base.cfg` is slightly outdated from the time before this option was added. I will add it and run some testing, then probably resolve this conversation and this PR can be closed.

Reply:
Suggestion was perfect, thanks!